loader/src/main/scala/com/actian/spark_vector/loader/parsers/Parser.scala

```diff
@@ -394,7 +404,15 @@ object Parser extends scopt.OptionParser[UserOptions]("spark-submit --class com.
         if (cur.extractor(options).isEmpty) acc :+ cur.longName else acc
       })
     if (missing.isEmpty) {
-      success
+        val instance = "[a-zA-Z]+".r
+        val offset = "[0-9]+".r
+        val port = "[a-zA-Z]*[0-9]+".r
```

Comment:
dumb question:
why does the regular expression for:
- instance exclude numerical digits
- port allow alphabetical characters

so, maybe put examples in comments above the expressions for clarification